app/stac_api/serializers.py

```diff
@@ -546,6 +561,14 @@ def get_fields(self):
         fields['checksum:multihash'] = fields.pop('checksum_multihash')
         return fields
 
+    def create(self, validated_data):
+        asset = Asset.objects.create(**validated_data)
+        return asset
+
+    def update(self, instance, validated_data):
+        instance.save()
+        return instance
+
```

Comment:
Are these really needed ? Does the default implementation not sufficient ? When the default implementation works I would rely on it.

Reply:
unit test succeed with the default implementation. So I deleted the explicit definitions here